class SchedulerAlreadyRunningError(Exception):
    """Raised when attempting to start or configure the speed_test when it's already running."""

    def __str__(self):
        return "Scheduler is already running"


class SchedulerNotRunningError(Exception):
    """Raised when attempting to shutdown the speed_test when it's not running."""

    def __str__(self):
        return "Scheduler is not running"
